docs/api_lib_index.md

```diff
@@ -5,6 +5,12 @@ sidebar_label: "Introduction"
 
 One of the foundational rules of Home Assistant is that we do not include any protocol specific code. Instead, this code should be put into a standalone Python library and published to PyPI. This guide will describe how to get started with this!
 
+## Basic library requirements
+- The library must have source distribution packages available<> it's not allowed to rely on packages that only have binary distribution packages.
```

Comment:
```suggestion
- The library must have source distribution packages available<> it's not allowed to rely on packages that only have binary distribution packages.
```
Does this mean that the packages also publish their source to PyPI? Didn't know this was a thing

Reply:
Yes, pypi allows publishing both, a wheel contain platform-specific, prebuilt packages (i.e., something that you can just unpack to install) while source distributions requires building it (e.g., using the setup.py) as part of the installation.

You usually publish both source and binary distributions, as wheels avoid the need to build from the source, and can be seen on packages which use native code, for example https://pypi.org/project/numpy/#files . Wheels are usually preferred as you don't need to install the build toolchains just to use the package.